salt/orch/kubernetes.sls

```diff
@@ -192,3 +192,13 @@ clear_bootstrap_in_progress_flag:
       - false
     - require:
       - salt: set_bootstrap_complete_flag
+
+# Ensure that /etc/hosts on velum is updated, bsc#1062728
+update_velum_hosts:
+  salt.state:
+    - tgt: 'roles:admin'
+    - tgt_type: grain
+    - sls:
+      - etc-hosts.velum
+    - require:
+      - salt: clear_bootstrap_in_progress_flag
```

Comment:
I wonder why this is at the end of this orchestration, I think it would also make a lot of sense to completely remove this from this orchestration and move it to the https://github.com/kubic-project/salt/blob/master/salt/orch/update-etc-hosts.sls orchestration, that way  we are sure it's kept in sync with possible ip changes.

Reply:
I've added also this step into `salt/orch/update-etc-hosts.sls` but here we have to keep it as well. It works only like this.